frontend/src/features/documents/documentDetail/DocumentDetailView.tsx

```diff
@@ -53,29 +78,125 @@ const DocumentDetailView: React.FunctionComponent<IDocumentDetailsViewProps> = p
           <StyledGreySection>
             <Row className="pb-3">
               <Col className="text-left">
-                <StyledH2>Document information</StyledH2>
+                <StyledHeader>
+                  <StyledH2>Document information</StyledH2>
+                  <TooltipIcon
+                    toolTipId={'documentInfoToolTip'}
+                    className={'documentInfoToolTip'}
+                    toolTip="Information you provided here will be searchable"
+                  ></TooltipIcon>
+                </StyledHeader>
               </Col>
-              {hasClaim(Claims.DOCUMENT_EDIT) && <Col xs="2">Edit</Col>}
-            </Row>
-            <SectionField label="Status" labelWidth="4">
-              {props.document.pimsDocument?.statusTypeCode?.description}
-            </SectionField>
-
-            <StyledH3>Details</StyledH3>
-            <StyledScrollable>
-              {props.document.mayanMetadata?.length === 0 && (
-                <StyledNoData>No additional data</StyledNoData>
+              {hasClaim(Claims.DOCUMENT_EDIT) && (
+                <Col xs="2">
+                  {' '}
+                  <LinkButton
+                    onClick={() => {
+                      setEditable(true);
+                    }}
+                  >
+                    <FaEdit />
+                  </LinkButton>
+                </Col>
               )}
-              {props.document.mayanMetadata?.map(value => (
-                <SectionField
-                  labelWidth="4"
-                  key={`document-${value.document.id}-metadata-${value.id}`}
-                  label={value.metadata_type.label || ''}
-                >
-                  {value.value}
+            </Row>
+            {!editable && (
+              <>
+                <SectionField label="Status" labelWidth="4">
+                  {props.document.pimsDocument?.statusTypeCode?.description}
                 </SectionField>
-              ))}
-            </StyledScrollable>
+                <StyledScrollable>
+                  {props.document.mayanMetadata?.length === 0 && (
+                    <StyledNoData>No additional data</StyledNoData>
+                  )}
+                  {props.document.mayanMetadata?.map(value => (
```

Comment:
Do we not already have similar code for the create screen? If so we should refactor this into a common component so that if we need to make changes to the metadata logic we don't have to do it in two places.